felix_test.py

```diff
@@ -0,0 +1,61 @@
+import sys
+sys.path.append('/home/icb/anna.chernysheva/epytope')
```

Comment:
Can you make this relative (e.g. sys.path.append(".."))? Then I can also use it on the cluster